awx/main/migrations/0050_v340_drop_celery_tables.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0049_v330_validate_instance_capacity_adjustment'),
+    ]
+
+    operations = [
+        migrations.RunSQL([
+            ("DROP TABLE IF EXISTS {} CASCADE;".format(table))
+        ])
+        for table in ('celery_taskmeta', 'celery_tasksetmeta', 'djcelery_crontabschedule',
```

Comment:
@matburt any concerns over the performance/speed of dropping any of these tables on upgrade?  _maybe_ `celery_taskmeta`?  My understanding is that celery periodically cleans these, so I don't expect _too much_ by way of table size.